source/common/tracing/zipkin/span_buffer.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "common/tracing/zipkin/zipkin_core_types.h"
+
+namespace Zipkin {
+
+/**
+ * This class implements a simple buffer to store Zipkin tracing spans
+ * prior to flushing them.
+ */
+class SpanBuffer {
+public:
+  /**
+   * Constructor that creates an empty buffer. Space needs to be allocated by invoking
+   * the method allocateBuffer(size).
+   */
+  SpanBuffer() {}
```

Comment:
Yes, this is used by the code on PR #700. PR #700, which is supposed to be merged after this, makes use of the classes on this PR to perform the actual Zipkin tracing.